src/components/DragDrop/DragDrop.module.css

```diff
@@ -0,0 +1,86 @@
+/* TODO: rm these once eslint change is in next */
```

Comment:
the lint change should be in btw. if any lines are still throwing errors, you can disable those specific lines with
`/* stylelint-disable-next-line max-nesting-depth */`